src/graphql/client.ts

```diff
@@ -77,6 +77,9 @@ export const apolloClient = new ApolloClient({
           },
         },
       },
+      App: {
+        keyFields: false,
+      },
```

Comment:
ok, so this way we will be disabling normalization of apps (by default they would be stored as `App:<id>` in Apollo Cache.

I'm wondering if this change is actually hiding other underlying issue in dashboard, where some mutation changes app and breaks the cache instead? I don't quite understand how this fixes a loading issue when fetching apps list 🤔

Reply:
it's jus change the way of normalisation, tl;dr now query results are stored separately, without impacting each other, sometimes it make sense.

Generally speaking, updating apollo would help in the future since as i recall there are more settings related to the cache and you can change something from the query level, without touching global config. eg when in certain case you want to update something